### ISSUE TITLE
CanCOGeN and MPXV templates: fix rule for transforming Homo sapiens to Human in NML LIMS output

### ISSUE THREAD

**Issue by griffie** [cidgoh/pathogen-genomics-package#11]
There are 2 places in the NML LIMS export that the field "host (scientific name)" outputs to - the field that goes into NML LIMS called PH_SPECIMEN_SOURCE and the DH field that also appears in the export file but doesn't get uploaded to LIMS called "host (scientific name)".

In the recent changes to the PGP, we lost the rule that says IF host (scientific name) is Homo sapiens THEN PH_SPECIMEN_SOURCE is Human. The NML uses "Human" instead of "Homo sapiens".

The issue we had before was that the DH is outputting the Human rule in the host (scientific name) field as well as PH_SPECIMEN_SOURCE. 
i.e. IF host (scientific name) is Homo sapiens THEN host (scientific name) should be Homo sapiens and NOT Human. 

In other words, we want the entered data (Homo sapiens) to be in the DH output fields (lower case after the Provenance field), but the transformed value (Human) in the NML LIMS field (PH_SPECIMEN_SOURCE, before the Provenance field). 

Can we do this?
The fix is needed for the NML LIMS from both the CanCOGeN and Monkeypox templates.
